package.json

```diff
@@ -59,7 +59,8 @@
     "reboot": "yarn clean && yarn reset && yarn install && yarn start",
     "rollup.bundle": "rollup -c -m",
     "rollup.watch": "yarn rollup -w",
-    "rollup": "yarn rollup.bundle"
+    "rollup": "yarn rollup.bundle",
+    "test": "tsc && wtr"
```

Comment:
TIL you can comment the `package.json` with `//` keys.

https://stackoverflow.com/a/14221781

Might be helpful for short named things like the TypeScript compiler and Web Test Runner?

`"//": "Run the TypeScript compiler and the Web Test Runner.",`

Reply:
I get yelled at when I try to do this

![Screen Shot 2021-07-19 at 10 33 14 AM](https://user-images.githubusercontent.com/31560031/126202205-0446aad8-cb09-49e7-a449-ddd279f4c002.png)

Reply:
You can't add a comment at the end of a line, the key needs to be `//` as if it were another dependency / command / property on the object. It is confusing. :(

Reply:
oh interesting